api/src/main/java/org/ehcache/spi/service/PersistableResourceService.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.spi.service;
+
+import org.ehcache.config.ResourcePool;
+import org.ehcache.config.ResourceType;
+import org.ehcache.exceptions.CachePersistenceException;
+import org.ehcache.spi.ServiceProvider;
+
+import java.util.Collection;
+
+/**
+ * Specific interface for services that are dedicated to handling a {@link ResourceType} which is
+ * {@link ResourceType#isPersistable() persistable}.
+ */
+public interface PersistableResourceService extends Service {
+
+  /**
+   * Start this service for maintenance, based on its default configuration.
+   */
+  void startForMaintenance();
```

Comment:
I'm in favor of decoupling the "persistability" from the "maintainability". But just renaming the interface to something like `MaintainableService` should suffice IMHO.